test/python/http_recorder.py

```
@@ -0,0 +1,285 @@
+            be dump in cassette_path, using serializer.
+            serializer (callable): the serializer for dumping cassette_dict in cassette_path.
+        """
+        ids2remove = {'api/users/loginWithToken': ['id',
+                                                   'userId',
+                                                   'created'],
+                      'api/Jobs': ['id',
+                                   'userId',
+                                   'creationDate',
+                                   'qasms.executionId',
+                                   'qasms.result.date',
+                                   'qasms.result.data.time',
+                                   'qasms.result.data.additionalData.seed'],
+                      'api/Backends/ibmqx5/queue/status': ['lengthQueue'],
+                      'api/Backends/ibmqx4/queue/status': ['lengthQueue']}
+        IdRemoverPersister.remove_ids(ids2remove, cassette_dict)
+        super(IdRemoverPersister, IdRemoverPersister).save_cassette(cassette_path,
+                                                                    cassette_dict,
+                                                                    serializer)
+
+
+def purge_headers(headers):
+    """
+    Remove headers from the response.
+
+    Args:
+        headers (list): headers to remove from the response
+
+    Returns:
+        callable: for been used in before_record_response VCR constructor.
+    """
+
+    header_list = list()
+    for item in headers:
+        if not isinstance(item, tuple):
+            item = (item, None)
+        header_list.append((item[0], item[1]))
+
+    def before_record_response(response):
+        """
+        Purge headers from response.
+
+        Args:
+            response (dict): a VCR response
+
+        Returns:
+            dict: a VCR response
+        """
+        for (header, value) in header_list:
+            try:
```

Use [`contextlib.supress(KeyError)`](
https://docs.python.org/3/library/contextlib.html#contextlib.suppress) to make it explicit that you want to ignore this error.

---

2e24709